tests/integration/tables/jetbrains_plugins.cpp

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2014-present, The osquery authors
+ *
+ * This source code is licensed as defined by the LICENSE file found in the
+ * root directory of this source tree.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR GPL-2.0-only)
+ */
+
+// Sanity check integration test for jetbrains_plugins
+// Spec file: specs/jetbrains_plugins.table
+
+#include <osquery/tests/integration/tables/helper.h>
+#include <osquery/tests/test_util.h>
+
+namespace osquery {
+namespace table_tests {
+class jetbrainsPlugins : public testing::Test {
+ protected:
+  void SetUp() override {
+    setUpEnvironment();
+  }
+#ifdef OSQUERY_WINDOWS
+  static void SetUpTestSuite() {
+    initUsersAndGroupsServices(true, false);
+  }
+
+  static void TearDownTestSuite() {
+    Dispatcher::stopServices();
+    Dispatcher::joinServices();
+    deinitUsersAndGroupsServices(true, false);
+    Dispatcher::instance().resetStopping();
+  }
+#endif
```

Comment:
Not having this block was making the windows test fail. I see that `usersFromContext(context)` fails without it.
```
80: E20250611 01:13:27.886477  7396 registry_factory.cpp:188] table registry users plugin caused exception: no state
80: E20250611 01:13:27.886477  7396 virtual_table.cpp:1052] Exception while executing table jetbrains_plugins: no state
```